features/openapi/docs.feature

```diff
@@ -432,3 +432,9 @@ Feature: Documentation support
     ]
     """
     And the JSON node "components.schemas.DummyBoolean.properties.isDummyBoolean.owl:maxCardinality" should not exist
+
+  Scenario: Retrieve the OpenAPI documentation in JSON
+    Given I add "Accept" header equal to "text/html,*/*;q=0.8"
```

Comment:
@bendavies note that browsers are sending `*/*` without that we always control the Accept header before the `.{_format}` and we'd get a 406.

Reply:
@soyuka i'm not sure what you mean sorry.